pkg/cmd/roachprod/flags.go

```diff
@@ -208,13 +208,13 @@ func initFlags() {
 		`Recurrence and scheduled backup options specification.
 Default is "RECURRING '*/15 * * * *' FULL BACKUP '@hourly' WITH SCHEDULE OPTIONS first_run = 'now'"`)
 
-	startTenantCmd.Flags().StringVarP(&hostCluster,
-		"host-cluster", "H", "", "host cluster")
-	_ = startTenantCmd.MarkFlagRequired("host-cluster")
+	startTenantCmd.Flags().StringVarP(&storageCluster,
+		"storage-cluster", "H", "", "storage cluster")
+	_ = startTenantCmd.MarkFlagRequired("storage-cluster")
 	startTenantCmd.Flags().IntVarP(&startOpts.TenantID,
```

Comment:
Is `tenant-id` the name we use to refer to this? I'm thinking about the number of "concepts" involved in the current `start-tenant` command: tenant itself, `tenant-id`, virtual cluster, storage cluster, sql instance. Do we want to eventually replace `tenant-id` with `virtual-cluster-id` or are they separate things?